core/src/test/scala/integration/kafka/api/EndToEndAuthorizationTest.scala

```diff
@@ -201,6 +212,22 @@ abstract class EndToEndAuthorizationTest extends IntegrationTestHarness with Sas
     consumeRecords(this.consumers.head, numRecords)
   }
 
+  @Test
+  def testProduceConsumeWithWildcardAcls(): Unit = {
+    setWildcardAclsAndProduce()
```

Comment:
Was there a good reason to couple the ACL updates and the producing? I think it would be a little clearer to keep them separate.

Reply:
@hachikuji  Thanks for the review. Update the PR.